runtime/descriptor.cpp

```diff
@@ -27,11 +28,13 @@ Descriptor::~Descriptor() {
 void Descriptor::Establish(TypeCode t, std::size_t elementBytes, void *p,
     int rank, const SubscriptValue *extent, ISO::CFI_attribute_t attribute,
     bool addendum) {
-  CHECK(ISO::CFI_establish(&raw_, p, attribute, t.raw(), elementBytes, rank,
-            extent) == CFI_SUCCESS);
+  Terminator terminator{__FILE__, __LINE__};
+  RUNTIME_CHECK(terminator,
+      ISO::CFI_establish(&raw_, p, attribute, t.raw(), elementBytes, rank,
+          extent) == CFI_SUCCESS);
```

Comment:
`CHECK` in the compiler proper always executes its argument, and if `RUNTIME_CHECK` does not do so, I'll fix it.

Reply:
To confirm: `RUNTIME_CHECK` will execute its argument.

Reply:
I think it does indeed, I was just thinking someone might come in runtime/terminator.h and disable `RUNTIME_CHECK` in release mode thinking it is only debug related because it is a macro with `CHECK` in its name. They would have a weird surprise here if they did not audit the use sites of  `RUNTIME_CHECK`.

Reply:
I'll add a comment.  I'm not sure what more can be done.

Reply:
Sounds enough to me.